sharkclipper/api/handlers.py

```diff
@@ -91,7 +92,10 @@ def save(handler, path, temp_dir = None, data = None, out_dir = None, **kwargs):
     # Out directory name is video name plus prefix of id.
     id_prefix = data['video']['id'].split('-')[0]
     out_dir = os.path.join(out_dir, data['video']['name'] + '-' + id_prefix)
-    os.makedirs(out_dir, exist_ok = True)
+    # If directory already exists, remake to avoid conflicts.
+    if os.path.exists(out_dir):
+        shutil.rmtree(out_dir)
```

Comment:
I would put a newline after this.
This follows along with a comment I just made in another of your PRs about usually not having stuff after the end of a conditional (in the same visual block).

Reply:
I will try and take note of these whitespace rules going forward.